Library/Homebrew/os/mac/version.rb

```diff
@@ -6,7 +6,7 @@ module OS
   module Mac
     class Version < ::Version
       SYMBOLS = {
-        big_sur:     "10.16",
+        big_sur:     "11.00",
```

Comment:
`MacOS.version` returns 10.16 on Intel (for now) so any comparisons will not work.

Reply:
This is just for converting the symbol to the version so I think it's still the right thing here. We'll likely need some custom comparison logic too to know that 10.16 == 11.0.

Reply:
This also affects Cask definitions, which currently use the `>=` operator for `depends_on` ([example](https://github.com/Homebrew/homebrew-cask/blob/8b10339f0e06c23f261b06fc358baff64bd23869/Casks/wwdc.rb#L12)).

/cc @vitorgalvao

Reply:
Made it conditional on ARM/Intel for now.